### PR TITLE
check admin: check store has pending admin reqeusts

### PR DESCRIPTION
**Pull request by 3pointer**

### CODE REVIEW

**Review by hicqu — APPROVED** [review]

File: proto/brpb.proto [hunk 1]
```diff
@@ -347,8 +348,21 @@ message BackupResponse {
...
+    metapb.Region region = 2;
+    bool has_pending_admin = 3;
+}
+
 service Backup {
     rpc backup(BackupRequest) returns (stream BackupResponse) {}
+    // CheckAdmin used for snapshot backup. before we start snapshot for a TiKV. 
+    // we need stop all schedule first and make sure all in-flight schedule has finished.  
+    // this rpc check all pending conf change for leader.
+    rpc CheckAdmin(CheckAdminRequest) returns (stream CheckAdminResponse) {}
```
Comment (D3Hunter) [inline proto/brpb.proto:365]: name it to HasRegionSchedule?

Comment (3pointer) [reply proto/brpb.proto:365]: I think schedule is also not clear, we need check split/merge rather than schedule 🤔
CheckPendingAdmin maybe better?

Comment (D3Hunter) [reply proto/brpb.proto:365]: or `CheckPendingAdminOp`, don't know what it mean by `pending admin`

Comment (3pointer) [reply proto/brpb.proto:365]: CheckPendingAdminOp is ok for me, WDYT? cc @hicqu

Comment (3pointer) [reply proto/brpb.proto]: change it to CheckPendingAdminOp for now, if it's not ok, we can change it in future.

**Review by D3Hunter — APPROVED** [review]

**Pull request merged**